Encountify.Android/Encountify.Android.csproj

```diff
@@ -69,23 +69,28 @@
     <PackageReference Include="Xam.Plugin.Geolocator">
       <Version>4.5.0.6</Version>
     </PackageReference>
-    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2125" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2196" />
     <PackageReference Include="Xamarin.Essentials" Version="1.7.0" />
-    <PackageReference Include="Xamarin.Android.Support.Compat" Version="26.0.2" />
-    <PackageReference Include="Xamarin.Android.Support.Core.UI" Version="26.0.2" />
-    <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="26.0.2" />
-    <PackageReference Include="Xamarin.Android.Support.Fragment" Version="26.0.2" />
-    <PackageReference Include="Xamarin.Android.Support.Media.Compat" Version="26.0.2" />
+    <PackageReference Include="Xamarin.Android.Support.Compat" Version="28.0.0.3" />
+    <PackageReference Include="Xamarin.Android.Support.Core.UI" Version="28.0.0.3" />
+    <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="28.0.0.3" />
+    <PackageReference Include="Xamarin.Android.Support.Fragment" Version="28.0.0.3" />
+    <PackageReference Include="Xamarin.Android.Support.Media.Compat" Version="28.0.0.3" />
```

Comment:
For me project does not build, unless I undo changes to lines 74-78. After change project seems to be working fine